tests/scripts/run_python_model_tests.sh

```diff
@@ -42,6 +42,17 @@ run_python_model_tests_wormhole_b0() {
     # Mamba
     WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest -svv models/demos/wormhole/mamba/tests/test_residual_block.py -k "pretrained_weight_false"
 
-    # Llama 3.1 8B single-layer dummy weights tight PCC check
-    WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml pytest models/demos/wormhole/llama31_8b/tests/test_llama_model.py -k "quick"
+    wh_arch_yaml=wormhole_b0_80_arch_eth_dispatch.yaml
```

Comment:
This isn't reused. Maybe just have `WH_ARCH_YAML=wormhole_b0_80_arch_eth_dispatch.yaml` instead of `WH_ARCH_YAML=wh_arch_yaml`